package.json

```diff
@@ -8,7 +8,8 @@
     "start": "babel-node config/bridges.js",
     "rpm-builder:setup": "docker build -t rippled-rpm-builder rpm-builder/",
     "rpm-deployer:setup": "docker build -t rippled-rpm-deployer rpm-deployer/",
-    "rpm-tester:setup": "docker build -t rippled-rpm-tester rpm-tester/"
+    "rpm-tester:setup": "docker build -t rippled-rpm-tester rpm-tester/",
+    "ansible:deploy:staging": "ansible-playbook -i ansible/hosts -l staging --extra-vars='@ansible/vault.yml' --ask-vault-pass ansible/bot.yml"
```

Comment:
what if `ansible:deploy:staging` had `cd ansible && ...`, then we could keep `ansible_ssh_private_key_file` as `skynet-keys/ap-southeast-1.pem` in the hosts file